jetty-deploy/src/main/config/etc/jetty-deploy.xml

```diff
@@ -55,6 +55,7 @@
               </Set>
               <Set name="scanInterval"><Property name="jetty.deploy.scanInterval" default="1"/></Set>
               <Set name="extractWars"><Property name="jetty.deploy.extractWars" default="true"/></Set>
+              <Set name="stopOnWebappFail"><Property name="jetty.deploy.stopOnWebappFail" default="false"/></Set>
```

Comment:
Name would be better as `stopOnFailedDeploy` or `stopOnFailedContext` as it is not always a Webapp.

Reply:
good point, done